source/js/foundation/pages/youtube-regrets/index.js

```diff
@@ -11,22 +11,29 @@ const BLOCK_SPEED_FACTOR = 0.8;
 const RING_SPEED_FACTOR = 0.6;
 
 let elements = {
```

Comment:
I wanted to rename this file to `tunnel.js` but wanted to do that after the code review as renaming a file can make code review harder.